ios/Classes/NativeBridge/Handlers/AuthenticatorsHandler.swift

```diff
@@ -84,7 +54,13 @@ extension AuthenticatorsHandler: BridgeToAuthenticatorsHandlerProtocol {
     }
 }
 
-extension AuthenticatorsHandler: AuthenticatorRegistrationDelegate {
+class AuthenticatorRegistrationDelegateImpl: AuthenticatorRegistrationDelegate {
+    private var registrationCompletion: ((Result<Void, FlutterError>) -> Void)?
```

Comment:
I think the name `completion` is enough - you don't have another block within this class, do you?